web/src/features/charts/BreakdownChart.tsx

```diff
@@ -72,14 +73,15 @@ function BreakdownChart({
         icon={<CircleBoltIcon />}
         unit={valueAxisLabel}
       />
-      <div className="relative">
+      <div className="relative ">
         {isBreakdownGraphOverlayEnabled && (
-          <div className="absolute top-0 h-full w-full">
-            <div className=" h-full w-full bg-white opacity-50 dark:bg-gray-800" />
-            <div className="absolute left-1/2 top-1/2 z-10 -translate-x-1/2 -translate-y-1/2 whitespace-nowrap rounded-sm bg-gray-200 p-2 text-center text-sm shadow-lg dark:bg-gray-900">
-              Temporarily disabled for consumption. <br /> Switch to production view
-            </div>
-          </div>
+          <DisabledMessage
+            message={
+              <>
+                Disabled for consumption. <br /> Switch to production view
```

Comment:
```suggestion
                Disabled for consumption view. <br /> Please switch to production view to enable the origin chart
```
or similar?

Reply:
Should this not be a translation string so it can well be translated if we are going to change it anyway?